### DISCUSSION TITLE
How "granular" should be the title of an Activity?

### DISCUSSION THREAD

**Discussion by trallnag** [kimai/kimai#2738] [Questions & Answers] [answered]
## Context

Currently trying out different time tracking tools for personal usage. Mainly looking at Kimai and Clockify.

## Question

In Clockify there are clients, projects, and simply free text descriptions. So if I start working on something I for example add the item in the calendar view and give it a name like:

* Research better tool for time management
* Try out Python lib Dynaconf
* Fix bug XYZ

Is it the same for Kimai? Since in Kimai I have to create new activities on a separate pane it seems like they are supposed to be more high level. For example:

* Research
* Bug Fixing

Is my assumption correct?

**Comment by kevinpapst** [comment, marked as answer]
Yes. Need more infos 😁 ?

Activities will be made optional (by configuration) in the future. Some companies use them a lot, especially for reporting. Some don't need them at all.

As freelancer, you might be more of the second type and not really need them. Personally, I have a handful of global activities like Development, DevOps, Consulting, Meeting ... makes it simpler when giving timesheets to customers or when grouping invoice items (in case you want to use Kimai for invoicing).

**Reply by trallnag** [reply]
ok thanks
